commands/profile_event_uploads.go

```diff
@@ -104,6 +106,14 @@ func isIdentity(val string) bool {
 	return false
 }
 
+func isIdentityValid (val string, pattern string) bool {
+	if(pattern == "") {
+		return true
+	}
+	re := regexp.MustCompile(pattern)
```

Comment:
`regexp.MustCompile(pattern)` 
Can this be done once in some global context, so you don't need to compile pattern for every call to isIdentityValid